applications/FluidDynamicsApplication/python_scripts/apply_embedded_skin_visualization_process.py

```diff
@@ -12,100 +12,42 @@ def __init__(self, Model, settings):
+            KratosMultiphysics.Logger.PrintWarning('\n\x1b[1;31mDEPRECATION-WARNING\x1b[0m', warn_msg)
+        if settings.Has("output_configuration"):
+            settings.RemoveValue("output_configuration")
+            warn_msg = "\"output_configuration\" is no longer needed as the visualization mesh is no longer print in the \"ApplyEmbeddedSkinVisualizationProcess\". Removing from input settings.\n"
+            warn_msg += "Add your preferred output process in the \"output_process_list\" of the simulation settings (namely ProjectParameters.json)."
+            KratosMultiphysics.Logger.PrintWarning('\n\x1b[1;31mDEPRECATION-WARNING\x1b[0m', warn_msg)
+
+        # Create the embedded skin visualization proces
         self.EmbeddedSkinVisualizationProcess = KratosFluid.EmbeddedSkinVisualizationProcess(
-            self.origin_model_part,
-            self.visualization_model_part,
-            aux_params)
-
-        # Set the output variables and build the GiD output process
-        if (settings["parallel_type"].GetString() == "OpenMP"):
-            from KratosMultiphysics.gid_output_process import GiDOutputProcess
-            self.gid_output = GiDOutputProcess(self.visualization_model_part, settings["visualization_model_part_name"].GetString(), settings["output_configuration"])
-        elif (settings["parallel_type"].GetString() == "MPI"):
-            from KratosMultiphysics.mpi.distributed_gid_output_process import DistributedGiDOutputProcess
-            self.gid_output = DistributedGiDOutputProcess(self.visualization_model_part, settings["visualization_model_part_name"].GetString(), settings["output_configuration"])
+            Model,
+            settings)
 
     def ExecuteInitialize(self):
-        self.gid_output.ExecuteInitialize()
         self.EmbeddedSkinVisualizationProcess.ExecuteInitialize()
 
     def ExecuteBeforeSolutionLoop(self):
         self.EmbeddedSkinVisualizationProcess.ExecuteBeforeSolutionLoop()
-        self.gid_output.ExecuteBeforeSolutionLoop()
 
     def ExecuteInitializeSolutionStep(self):
         self.EmbeddedSkinVisualizationProcess.ExecuteInitializeSolutionStep()
-        self.gid_output.ExecuteInitializeSolutionStep()
 
     def ExecuteFinalizeSolutionStep(self):
         self.EmbeddedSkinVisualizationProcess.ExecuteFinalizeSolutionStep()
-        self.gid_output.ExecuteFinalizeSolutionStep()
 
     def ExecuteBeforeOutputStep(self):
         self.EmbeddedSkinVisualizationProcess.ExecuteBeforeOutputStep()
-        if (self.gid_output.IsOutputStep()):
-            self.gid_output.PrintOutput()
 
     def ExecuteAfterOutputStep(self):
         self.EmbeddedSkinVisualizationProcess.ExecuteAfterOutputStep()
-        self.gid_output.ExecuteAfterOutputStep()
 
     def ExecuteFinalize(self):
         self.EmbeddedSkinVisualizationProcess.ExecuteFinalize()
-        self.gid_output.ExecuteFinalize()
+
+    def Check(self):
```

Comment:
Instead of overridding all those functions I would return the C++ process in the `Factory` method of this process, this way you make sure all the methods are called
See e.g. [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/FluidDynamicsApplication/python_scripts/integration_point_statistics_process.py) and [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py)

Reply:
Cool. I'll do that.

Reply:
Done. Thanks for the hint.